docs/index.html

```diff
@@ -51,6 +51,20 @@ <h2 class="callout-subtitle">Just-add-water CSS animations</h2>
 
       <section class="animation-list">
         <button class="callout-hideList button">Close list</button>
+
+        <a class="dark-mode-button-container">
```

Comment:
You have to make these changes in the [`docsSource/template.html`](https://github.com/animate-css/animate.css/blob/main/docsSource/template.html) file.

You'll also need to compile the docs using `npm run docs`.